### PR TITLE
feat: Configure tools application registry category - EXO-61480

### PR DESCRIPTION
**Pull request by sofyenne**
This change is going to add the agenda timeline under the tools application category

### CODE REVIEW

File: agenda-webapps/src/main/webapp/WEB-INF/conf/agenda/application-registry-configuration.xml [hunk 1]
```diff
@@ -89,5 +89,50 @@
...
+              <field name="applications">
+                <collection type="java.util.ArrayList">
+                  <value>
+                    <object type="org.exoplatform.application.registry.Application">
+                      <field name="applicationName"><string>AgendaTimeline</string></field>
+                      <field name="categoryName"><string>tools</string></field>
+                      <field name="displayName"><string>Agenda Timeline</string></field>
+                      <field name="description"><string>Agenda Timeline Portlet</string></field>
+                      <field name="type"><string>portlet</string></field>
+                      <field name="contentId"><string>agenda/AgendaTimeline</string></field>
```
Comment (azayati) [inline agenda-webapps/src/main/webapp/WEB-INF/conf/agenda/application-registry-configuration.xml]: separate the tags

File: agenda-webapps/src/main/webapp/WEB-INF/conf/agenda/application-registry-configuration.xml [hunk 2]
```diff
@@ -89,5 +89,50 @@
...
+                      <field name="accessPermissions">
+                        <collection type="java.util.ArrayList" item-type="java.lang.String">
+                          <value><string>*:/platform/users</string></value>
```
Comment (azayati) [inline agenda-webapps/src/main/webapp/WEB-INF/conf/agenda/application-registry-configuration.xml]: separate the tags

**Review by azayati — CHANGES_REQUESTED** [review]

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**